elmslie-plugin/src/main/resources/templates/PresentationReducer.kt.ftl

```diff
@@ -0,0 +1,47 @@
+package ${presentation_package_name}
+
+import vivid.money.elmslie.core.store.Result
+import vivid.money.elmslie.core.store.StateReducer
+import ${presentation_package_name}.${domain_name}Command.*
+<#if split_events>
+import ${presentation_package_name}.${domain_name}Event.Internal
+import ${presentation_package_name}.${domain_name}Event.Ui
+<#else>
+import ${presentation_package_name}.${domain_name}Event.*
+</#if>
+
+internal object ${domain_name}Reducer : StateReducer<${domain_name}Event, ${domain_name}State,
+        ${domain_name}Effect, ${domain_name}Command> {
```

Comment:
Since we have a DslReducer and a ScreenDslReducer, maybe it's better to use them in plugin?